src/main/java/racing/model/Car.java

```diff
@@ -0,0 +1,41 @@
+package racing.model;
+
+public class Car {
+    private String carName = "";
+    private int distance = 0;
+    private final static int MAX_WORD_NAME = 5;
+    public Car(String name) {
+        checkCarName(name);
+        this.carName = name;
+    }
+
+    public Car(String name, int distance) {
+        this.carName = name;
+        this.distance = distance;
+    }
+
+    public void go() {
+        this.distance = ++this.distance;
+    }
+
+    public int totalDistance() {
+        return this.distance;
+    }
+
+    public Boolean checkDistance(int distance) {
+        if (this.distance == distance) {
+            return true;
+        }
+        return false;
```

Comment:
```suggestion
        return this.distance == distance;
```
이렇게 요약해보면 좋을 것 같아요. :)